styleguide/_themes/global/scss/cta.scss

```diff
@@ -0,0 +1,40 @@
+.cta {
+  text-align: center;
+}
+
+.cta-headerWrapper {
+  @include make-md-column(12);
+}
+
+.cta-button {
+  @include button;
+  @include button-lead;
+  display: table;
+}
+
+.cta-buttonWrapper {
+  @include make-md-column(12);
+  margin-bottom: 3px;
+}
+
+.cta-iconWrapper {
+  @include make-md-column(12);
+}
+
+.cta-row {
+  @include make-row;
+}
+
+.cta-subheadWrapper {
+  @include make-md-column(12);
+
+  p {
```

Comment:
Being that there seem to be specific styles for this text, I think it is appropriate to have a specific class for it. Such a class can be in the CTA module, but something tells me that this combination of family, size and weight may be useful elsewhere. If not, it's a snowflake.